app/serializers/api/v1/active_model_errors_serializer.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require "cpdlp/model"
+
+module Api
+  module V1
+    class ActiveModelErrorsSerializer
+      include CPDLP::Model
+      include ActiveModel::Serialization
+
+      attribute :title
+      attribute :detail
+
+      def self.from(service)
+        {
+          errors: service
+            .errors
+            .messages
+            .map { |title, detail| new(title:, detail: detail.join(", ")).serializable_hash },
+        }
+      end
+    end
```

Comment:
This allows to render `all` the possible errors with given request. Before we could only render one at a time., due to the errors being handle with exceptions.